File: test-automation-core/src/main/java/com/nortal/test/core/services/report/html/PostmanDataReport.java
package com.nortal.test.core.services.report.html;

import java.util.List;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class PostmanDataReport {
	private final boolean enabled;
	private final List<CollectionData> collections;
	private final List<String> environments;

	@Data
	@Builder
	public static class CollectionData {
		private final String id;
		private final String name;
		private final String fileName;
	}
}

File: test-automation-core-old/src/main/java/com/nortal/test/core/services/report/html/PostmanDataReport.kt
package com.nortal.test.core.services.report.html

import lombok.Builder
import lombok.Data

@Data
@Builder
class PostmanDataReport {
    private val enabled = false
    private val collections: List<CollectionData>? = null
    private val environments: List<String>? = null

    @Data
    @Builder
    class CollectionData {
        private val id: String? = null
        private val name: String? = null
        private val fileName: String? = null
    }
}